platform/o.n.bootstrap/external/platform-launchers-12.3-license.txt

```diff
@@ -1,6 +1,6 @@
 Name: NetBeans Application Launchers
 Description: Windows Launchers for the NetBeans Platform
-Version: 10.0
+Version: 12.3
```

Comment:
Well, is it really necessary to implement time travel here as we are just about to release NetBeans 12.2. Where the number 12.3 is coming from? So is this intentional or mistype or some coincidence or something has happend that I'm not aware of?

Reply:
I was surprised by the `12.3` number as well. However I was surprised by the previous `10.0` version too. Then I concluded the number doesn't mean much as the release isn't going to use this `ZIP` file at all, but build from the sources.

Sváťa may provide his explanation soon.

Reply:
Well, the launcher zip is only used when not building that from sources which is true almost always when not building a release. So I could understand that could be used for testing, still the last released launcher version is 12.1.
I'm going to go to sleep now, checking this PR in about 8 hours from now.